### ISSUE TITLE
Allow "watch groups" in debugger

### ISSUE THREAD

**Issue by zarakshR** [microsoft/vscode-cpptools#8716]
Type: Feature Request

For debugging the same set of variables over multiple different debug sessions, it would be nice if we could define groups of variables to load into the watch section of the debugger. This way we could delete the currently watched variables to keep the watch section clutter-free while also being able to load in a group of variables without having to add one manually.

An example of what I think something like this could look like -

I'm watching 6 different variables to narrow down a problem - v1, v2, v3, etc
I notice that the problem appears to be coming from somewhere else so I load in 4 other variables - x1, x2, etc
Now the watch section is too cluttered so I save v1, v2, etc in a group and delete them from the watch section so I can focus on the new ones.
After I'm done I can go back to watching the original 6 variables by clearing the watch section and loading back v1, v2, etc with one click from the group.

Basically tab groups but for watching debug variables

**Comment by WardenGnaw** [comment]
This would require a change on the VS Code side since this is applicable to other debuggers, too.

You will want to post to https://github.com/microsoft/vscode/pulls
